sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/enums/VehicleDataType.java

```diff
@@ -115,9 +115,13 @@
     VEHICLEDATA_MYKEY,
 
     VEHICLEDATA_FUELRANGE,
-    ;
 
-	/**
+    /**
+     * Notifies TURNSIGNALData may be subscribed
+     */
+    VEHICLEDATA_TURNSIGNAL;

```

Comment:
Can you put the semicolon on a new line with a new line in between? This will help reduce merge conflicts in the future and add less lines changed per addition.